README.md

```diff
@@ -1,5 +1,7 @@
 <h1 align="center">india-forecast-app</h1>
 
+[![ease of contribution: hard](https://img.shields.io/badge/ease%20of%20contribution:%20hard-bb2629)](https://github.com/openclimatefix/ocf-meta-repo?tab=readme-ov-file#how-easy-is-it-to-get-involved)
+
```

Comment:
I think this is easy actually, becasue it has fake data - see https://github.com/openclimatefix/ocf-meta-repo

Reply:
We have it down as hard/red in the meta repo but if we think this is fairly easy to contribute to then we can change that, personally I think it is more of an orange, you should be able to run it locally with a bit of setup for the data sources but the model HF repos are mostly open so can grab those

Reply:
oh sorry, i got confused between the india-api., yea hard red is good